tensorflow/compiler/tf2tensorrt/kernels/trt_engine_op_test.cc

```diff
@@ -153,6 +185,24 @@ class TRTEngineOpTestBase : public OpsTestBase {
   }
 };
 
+class TRTEngineOpTestWithParam
+    : public TRTEngineOpTestBase,
+      public ::testing::WithParamInterface<TestParam> {
+ public:
+  TRTEngineOpTestWithParam() : param_(GetParam()) {}
+
+ protected:
+  TestParam param_;
+};
+
+const absl::string_view TRTEngineOpTestBase::kOpName = "myop";
+
+constexpr std::array<TestParam, 2> TestParameters{TestParam{false, false},
```

Comment:
s/false, false/false, true/?

Reply:
We have two parameters one with `static_engine=false`, and another with `static_engine=true`. The first value `implicit_batch` is always false in the current tests, therefore I have removed it from TestParams (for now).